### ISSUE TITLE
Can I change the epoches of the train？

### ISSUE THREAD

**Issue by WujiangXu** [HuiZeng/Grid-Anchor-based-Image-Cropping#11]
I want to increase the epoches of the train or the times of the train in order to improve the training model of my dataset.Or are there any other ways to improve the training model.

**Comment by HuiZeng** [comment]
You can make any change in the matlab training code.
For pytorch code, I currently did not release the training code. You can implement it by yourself and make any change as your like. I will release the pytorch training code together with a journal version which has several improvements.

**Comment by WujiangXu** [comment]
Thank u for u answer.
I just use the code of matlab version.But I can't find the place to change to increase the times of the train or the epoches of the train.Whether it is set by the vgg16 or resnet50.

**Comment by HuiZeng** [comment]
This is a trivial implementation in the matconvnet.
You can change the setting of learning rate in the model initialization function. The epoch will automatically adjust to the length of the learning rate.
![image](https://user-images.githubusercontent.com/12780780/61268722-eb0b5200-a7ce-11e9-8ee7-5c1bcccf3366.png)

**Comment by WujiangXu** [comment]
thank you for your reply.I found it several minutes ago together.：）
